presto-docs/src/main/sphinx/admin/session-properties.rst

```diff
@@ -0,0 +1,25 @@
+==========================
+Session Properties Manager
+==========================
+
+Session properties manager provides a way to dynamically override default
+session properties. It supports overriding default session properties for each
+session based on ``SessionConfigurationContext``, which includes ``user``,
+``source`` and ``clientTags``. The returned properties override the default
+values but not the final values, thus any user-provided values will override
+the values returned by session properties manager.
+
+Implementation
+--------------
+
+``SessionPropertyConfigurationManagerFactory`` is responsible for creating an
+``SessionPropertyConfigurationManager`` instance. Implementations of
+``SessionPropertyConfigurationManager`` should include two methods:
+
+* ``getSystemSessionProperties``: Return a map of system session properties.
+
+* ``getCatalogSessionProperties``: Return a map of catalog to catalog properties map.
```

Comment:
Returns a map of catalog to catalog properties.

Reply:
`catalog to catalog properties map` sounds confusing.

Reply:
I agree. Not sure what's a better way to describe it without a long paragraph...